PersonStageThree/src/com/beyond/person/core/utils/enumeration/People.java

```diff
@@ -0,0 +1,27 @@
+package com.beyond.person.core.utils.enumeration;
+
+public enum People {
+    DANCER{
+        public String toString(){
+            return "1";
+        }
+    },
+
+    SINGER{
+        public String toString(){
+            return "2";
+        }
+    },
+
+    PROGRAMMER{
+        public String toString(){
+            return "3";
+        }
+    },
+
+    EXIT{
```

Comment:
@maria-salmasian people can have type like Exit ? ))))

Reply:
changed the name to inputChoice